### PR TITLE
APPDEV-9356 add base to audio import

### PR DESCRIPTION
**Pull request by sharonluong**
This PR adds the "base" column to audio imports.

### CODE REVIEW

File: app/Import/AudioImport.php [hunk 1]
```diff
@@ -154,6 +154,11 @@ public function validate()
           && !empty($row[$key]) && !$this->valueExists(AudioItem::class, 'track_configuration', $row[$key])) {
           $bag->add($key, $key . ' must already exist in the database.');
         }
+        // Validate track configuration exists in the DB
+        if ($key === 'Base'
+          && !empty($row[$key]) && !$this->valueExists(AudioItem::class, 'base', $row[$key])) {
+          $bag->add($key, $key . ' must already exist in the database.');
+        }
```
Comment (chrisrlc) [inline app/Import/AudioImport.php:161]: Doesn't need to be done in this PR, but could this code be DRY-ed up a bit? Looks mostly the same as the above couple of validations.

Comment (chrisrlc) [reply app/Import/AudioImport.php:161]: Hmm, actually, I guess it already was partly DRY-ed in the last PR I looked at, wasn't it. Still could be cleaned up a bit more with the similar check and messaging I suppose.

**Comment by chrisrlc** [comment]
@sharonluong - Looks good! Left a small comment that you can decide to do later or not. Up to you. If you decide to do later, this is ready to merge.

File: app/Import/AudioImport.php (hunk 1 above)
Comment (sharonluong) [reply app/Import/AudioImport.php:161]: Sure, that's a good point. I'll work on it as part of this PR since it's not a rush.

Comment (sharonluong) [reply app/Import/AudioImport.php:151]: Ok, the DRY-ed up code is up, can you take another look?

**Comment by chrisrlc** [comment]
Looks good, thanks for making that change!

**Pull request merged**